### PR TITLE
feat: add pt-br translations

### LINKED ISSUE #131: Brazilian Portuguese translation
hello, there! 
the PyLadies Brasil group was thinking about starting a Brazilian Portuguese translation of the cheatsheets and I wonder if there are specific guidelines for translations, i.e. where to save the translated files and so on. if not, can we fork and start? (:

### PR DESCRIPTION
**Pull request by giovana-morais**
Closes #131.

### CODE REVIEW

**Review by timhoffm — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Please do not overwrite the existing english versions.

Overall, we have to come up with a scheme how to manage translations:

- README: I think the best the pattern shown in https://github.com/tiimgreen/github-cheat-sheet: `README.[lang].md` files and a link *Read this in other languages* in the main README.
- Content: Please make new files for the translated handouts as well.

An unresolved problem is keeping translations synchronous and up to date.

**Comment by giovana-morais** [comment]
@timhoffm thanks for jumping in! 
I overwrote the file because I thought it would stay only in this fork, but I like your idea much better. I'll create the new files for both README and the handouts.

regarding keeping the translations open to date, I thought about maybe automatically creating an issue for the translation repository whenever we have new PRs merged in the official one, via the CI tool. not sure if it's the best approach, but I think it will give us visibility. :thinking:

File: README.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Folhas de dicas para usuários do Matplotlib
+
+## Observação
+O conteúdo original dessa folha de dicas é do repositório [oficial do
+`matplotlib`](https://github.com/matplotlib/cheatsheets). Essa tradução
+não-oficial visa tornar mais acessível as dicas para os usuários e usuárias
+do `matplotlib`.
```
Comment (melissawm) [inline README.pt-br.md:7]:
```suggestion
não-oficial visa tornar as dicas mais acessíveis para os usuários e usuárias
```
How about using gender-neutral language here? (I also am used to "a Matplotlib" as it is a library (uma biblioteca) but there's no right or wrong here 😅 

```suggestion
não-oficial visa tornar as dicas mais acessíveis para pessoas que usam a Matplotlib.
```

File: README.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+* `fonts/roboto-slab/*`      : See https://fonts.google.com/specimen/Roboto+Slab
+                                or https://github.com/googlefonts/robotoslab/tree/master/fonts/static
+* `fonts/source-code-pro/*`  : See https://fonts.google.com/specimen/Source+Code+Pro
+                                or https://github.com/adobe-fonts/source-code-pro/tree/release/OTF
+* `fonts/source-sans-pro/*`  : See https://fonts.google.com/specimen/Source+Sans+Pro
+                                or https://github.com/adobe-fonts/source-sans-pro/tree/release/OTF
+* `fonts/source-serif-pro/*` : See https://fonts.google.com/specimen/Source+Serif+Pro
+                                or https://github.com/adobe-fonts/source-serif-pro/tree/release/OTF
+* `fonts/eb-garamond/*`      : See https://bitbucket.org/georgd/eb-garamond/src/master
+* `fonts/pacifico/*`         : See https://fonts.google.com/download?family=Pacifico
```
Comment (melissawm) [inline README.pt-br.md:39]:
```suggestion
* `fonts/roboto/*`           : Veja https://fonts.google.com/specimen/Roboto
                                ou https://github.com/googlefonts/roboto/tree/master/src/hinted
* `fonts/roboto-slab/*`      : Veja https://fonts.google.com/specimen/Roboto+Slab
                                ou https://github.com/googlefonts/robotoslab/tree/master/fonts/static
* `fonts/source-code-pro/*`  : Veja https://fonts.google.com/specimen/Source+Code+Pro
                                ou https://github.com/adobe-fonts/source-code-pro/tree/release/OTF
* `fonts/source-sans-pro/*`  : Veja https://fonts.google.com/specimen/Source+Sans+Pro
                                ou https://github.com/adobe-fonts/source-sans-pro/tree/release/OTF
* `fonts/source-serif-pro/*` : Veja https://fonts.google.com/specimen/Source+Serif+Pro
                                ou https://github.com/adobe-fonts/source-serif-pro/tree/release/OTF
* `fonts/eb-garamond/*`      : Veja https://bitbucket.org/georgd/eb-garamond/src/master
* `fonts/pacifico/*`         : Veja https://fonts.google.com/download?family=Pacifico
```

Comment (melissawm) [inline README.pt-br.md:39]:
```suggestion
* `fonts/roboto/*`           : Veja https://fonts.google.com/specimen/Roboto
                                ou https://github.com/googlefonts/roboto/tree/master/src/hinted
* `fonts/roboto-slab/*`      : Veja https://fonts.google.com/specimen/Roboto+Slab
                                ou https://github.com/googlefonts/robotoslab/tree/master/fonts/static
* `fonts/source-code-pro/*`  : Veja https://fonts.google.com/specimen/Source+Code+Pro
                                ou https://github.com/adobe-fonts/source-code-pro/tree/release/OTF
* `fonts/source-sans-pro/*`  : Veja https://fonts.google.com/specimen/Source+Sans+Pro
                                ou https://github.com/adobe-fonts/source-sans-pro/tree/release/OTF
* `fonts/source-serif-pro/*` : Veja https://fonts.google.com/specimen/Source+Serif+Pro
                                ou https://github.com/adobe-fonts/source-serif-pro/tree/release/OTF
* `fonts/eb-garamond/*`      : Veja https://bitbucket.org/georgd/eb-garamond/src/master
* `fonts/pacifico/*`         : Veja https://fonts.google.com/download?family=Pacifico
```

File: README.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+No Linux, com o `make` instalado, as fontes podem ser configuradas com o seguinte comando:
+
+```shell
+make -C fonts
+```
+
+As fontes podem ser descobertas pelo `matplotlib` (por via do `fontconfig`) ao
+criar o seguinte `$HOME/.config/fontconfig/fonts.conf` (veja [aqui](https://www.freedesktop.org/software/fontconfig/fontconfig-user.html)):
```
Comment (melissawm) [inline README.pt-br.md:48]:
```suggestion
criar o arquivo `$HOME/.config/fontconfig/fonts.conf` com o seguinte conteúdo (veja [aqui](https://www.freedesktop.org/software/fontconfig/fontconfig-user.html)):
```

File: handout-beginner.pt-br.tex [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+% -----------------------------------------------------------------------------
+\begin{document}
+\thispagestyle{empty}
+
+\section*{\LARGE \rmfamily
+          Matplotlib \textcolor{orange}{\mdseries for beginners}}
+
+\begin{multicols*}{3}
+
+Matplotlib é uma biblioteca para gerar plots em 2D em Python. Foi desenhado
```
Comment (melissawm) [inline handout-beginner.pt-br.tex:74]:
```suggestion
Matplotlib é uma biblioteca para gerar plots em 2D em Python. Foi criada
```

File: handout-beginner.pt-br.tex [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+\end{lstlisting}
+& \raisebox{-0.75em}{\includegraphics[width=\linewidth]{advanced-boxplot.pdf}}
+\end{tabular}
+
+
+% -----------------------------------------------------------------------------
+\subsection*{\rmfamily Tweak}
+% -----------------------------------------------------------------------------
+Você pode modificar praticamente tudo em um plot, incluindo limites,
+cores, marcadores, grossura e estilo de linhas, ticks e rótulos de ticks,
```
Comment (melissawm) [inline handout-beginner.pt-br.tex:183]: Será que a gente consegue encontrar uma tradução para "ticks"?

File: handout-beginner.pt-br.tex [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+                           BoldFont       = Regular,
+                           BoldItalicFont = Italic ]
+\renewcommand{\familydefault}{\sfdefault}
+
+% -----------------------------------------------------------------------------
+\begin{document}
+\thispagestyle{empty}
+
+\section*{\LARGE \rmfamily
+          Matplotlib \textcolor{orange}{\mdseries for beginners}}
```
Comment (melissawm) [inline handout-beginner.pt-br.tex:70]:
```suggestion
          Matplotlib \textcolor{orange}{\mdseries para iniciantes}}
```

**Review by melissawm — COMMENTED** [review]
This is looking great, thank you so much @giovana-morais !

File: README.pt-br.md (hunk 2 above)
Comment (giovana-morais) [reply README.pt-br.md:39]: nossa, eu nem tinha visto esses pedaços hahahaha

File: README.pt-br.md (hunk 1 above)
Comment (giovana-morais) [reply README.pt-br.md:7]: por mim ok usar como "a matplotlib" no sentido de biblioteca. eu sempre pensei como "o módulo"

File: handout-beginner.pt-br.tex (hunk 5 above)
Comment (giovana-morais) [reply handout-beginner.pt-br.tex:183]: pois é, essa é uma questão que eu e a @R-it-a estávamos discutindo bastante e acho que você pode nos ajudar. tem alguns termos como "ticks", "axis" e "plot" que são muito ligados aos nomes das funções disponíveis na biblioteca. aí a nossa dúvida era: se a gente traduz, será que fica mais difícil de associar o nome ao comando?

**Comment by giovana-morais** [comment]
@timhoffm any suggestions on how to proceed with the Makefile? I thought about creating something like "make translation <language>", but I'm open to suggestions

File: README.pt-br.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+Façam bom uso!
+
+## Folhas de dicas
+Cheatsheet [(baixe o pdf)](https://matplotlib.org/cheatsheets/cheatsheets.pdf) | |
+:------------------------------------------------------------------------------:|:----------------------------------------------------------:
+![](https://matplotlib.org/cheatsheets/cheatsheets-1.png)                       | ![](https://matplotlib.org/cheatsheets/cheatsheets-2.png)
+
+## Folhetos
+
+Folheto iniciante [(download pdf)](https://matplotlib.org/cheatsheets/handout-beginner.pdf) | Folheto intermediário[(download pdf)](https://matplotlib.org/cheatsheets/handout-intermediate.pdf) | Folheto de dicas [(download pdf)](https://matplotlib.org/cheatsheets/handout-tips.pdf)
```
Comment (melissawm) [inline README.pt-br.md:17]:
```suggestion
Folheto iniciante [(download pdf)](https://matplotlib.org/cheatsheets/handout-beginner.pdf) | Folheto intermediário [(download pdf)](https://matplotlib.org/cheatsheets/handout-intermediate.pdf) | Folheto de dicas [(download pdf)](https://matplotlib.org/cheatsheets/handout-tips.pdf)
```

File: README.pt-br.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+:-----------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------:
+![](https://matplotlib.org/cheatsheets/handout-beginner.png)                               | ![](https://matplotlib.org/cheatsheets/handout-intermediate.png)                                   | ![](https://matplotlib.org/cheatsheets/handout-tips.png)
+
+# Para contribuídores das dicas
```
Comment (melissawm) [inline README.pt-br.md:21]:
```suggestion
# Para contribuidores das dicas
```

File: README.pt-br.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+## Como compilar
+
+1. Você precisa criar um repositório `fonts`com:
```
Comment (melissawm) [inline README.pt-br.md:25]:
```suggestion
1. Você precisa criar um repositório `fonts` com:
```

File: handout-beginner.pt-br.tex [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+% -----------------------------------------------------------------------------
+\begin{document}
+\thispagestyle{empty}
+
+\section*{\LARGE \rmfamily
+          Matplotlib \textcolor{orange}{\mdseries para iniciantes}}
+
+\begin{multicols*}{3}
+
+Matplotlib é uma biblioteca para gerar plots em 2D em Python. Foi criada
```
Comment (melissawm) [inline handout-beginner.pt-br.tex:74]: I am not sure if this is the original wording, but you can also do 3D plots in Matplotlib - maybe this should be updated?

File: handout-intermediate.pt-br.tex [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+\end{lstlisting}
+\includegraphics[width=\linewidth]{sine-annotate.pdf}
+
+% -----------------------------------------------------------------------------
+\subsection*{\rmfamily Cores}
+
+Qualquer cor pode ser usada, mas o Matplotlib oferece um conjunto de cores:\\
+\includegraphics[width=\linewidth]{colors-cycle.pdf} \smallskip
+\includegraphics[width=\linewidth]{colors-grey.pdf}\\
+%As well as nice colormaps (viridis an magma):\\
```
Comment (melissawm) [inline handout-intermediate.pt-br.tex:180]: Is this line also commented out in the original?

**Review by melissawm — APPROVED** [review]
Looking great - some minor comments but I'm approving since the contents in general are done. Obrigada @giovana-morais !!

File: handout-intermediate.pt-br.tex (hunk 11 above)
Comment (giovana-morais) [reply handout-intermediate.pt-br.tex:180]: yes, [it is!](https://github.com/matplotlib/cheatsheets/blob/898944be6f572ab82f03900b8b7e958e00c29c59/handout-intermediate.tex#L180-L182).

File: handout-beginner.pt-br.tex (hunk 10 above)
Comment (giovana-morais) [reply handout-beginner.pt-br.tex:74]: we followed the [original](https://github.com/matplotlib/cheatsheets/blob/898944be6f572ab82f03900b8b7e958e00c29c59/handout-beginner.tex#L74). should I add the 3D part in both documents?

Comment (melissawm) [reply handout-beginner.pt-br.tex:74]: That would be my suggestion!

**Comment by giovana-morais** [comment]
@timhoffm i was thinking that the specific Makefile for the translation handouts could be done in the PyLadies repo instead of here. can you review the PR as it is?

**Review by melissawm — APPROVED** [review]
@timhoffm as far as the text goes, I'm happy with it!

**Comment by giovana-morais** [comment]
I'll close this PR because I already merged the changes on the PyLadies Brasil fork. It makes more sense to keep it separate, so we can change the Makefile to generate the translated document instead of the original one and we have more freedom to use the translated version of the README as the original. 
@melissawm thank you for the help reviewing the translation and thanks @timhoffm for the guidance in the translation patterns! Later I can open a new PR with only an update on the README pointing to the PyLadies Brasil translation for Portuguese.

Obrigada!

**Pull request closed without merging**